tests/vm-migration

```diff
@@ -46,6 +46,17 @@ lxc exec member1 -- sh -c "$(declare -f waitSnapdSeed); waitSnapdSeed"
 # shellcheck disable=SC3044 # Ignore "declare is undefined" shellcheck error.
 lxc exec member2 -- sh -c "$(declare -f waitSnapdSeed); waitSnapdSeed"
 
+for instance in member1 member2; do
```

Comment:
it would be very helpful to add a comment here saying "workaround for LP: #2104066" otherwise it wont be clear whats happening for someone reading the script.

Reply:
oh wait you did further down